common/protob/messages-management.proto

```diff
@@ -280,6 +280,26 @@ message FirmwareHash {
     required bytes hash = 1;
 }
 
+/**
+ * Request: Request a signature of the provided challenge.
+ * @start
+ * @next AuthenticityProof
+ * @next Failure
+ */
+message AuthenticateDevice {
+    required bytes challenge = 1;  // A random challenge to sign.
+}
+
+/**
+ * Response: Signature of the provided challenge along with a certificate issued by the Trezor company.
+ * @end
+ */
+message AuthenticityProof {
+    required bytes ca_certificate = 1;      // Intermediate CA certificate signed by Trezor company's root CA.
+    required bytes device_certificate = 2;  // Device certificate signed by the intermediate CA, containing the public key that can be used to verify the challenge signature.
```

Comment:
wouldn't it be cleaner to do `repeated bytes certificates` ? in case we mess with the number of intermediates

Reply:
Seems reasonable to me. FYI @szymonlesisz.

Reply:
Done in https://github.com/trezor/trezor-firmware/pull/3255/commits/765e88b99a80c91ff470c5fc3d1ff0fac5b0533a, https://github.com/trezor/trezor-firmware/pull/3255/commits/1675a2658d7b9ae3f0b96bbd5f7413dc93e00ca1, https://github.com/trezor/trezor-firmware/pull/3255/commits/7b88826c217f9b755ef56db208b9f756d789d0af, https://github.com/trezor/trezor-firmware/pull/3255/commits/d20051510aba66b1f5bfd30ce464e18de2672e86.